php/libraries/NDB_BVL_Instrument.class.inc

```diff
@@ -270,11 +285,9 @@ abstract class NDB_BVL_Instrument extends NDB_Page
             $base . "project/instruments/$instrument.linst"
         );
         // Add rules only if they exist.
-        if (file_exists($base."project/instruments/$instrument.rules")) {
```

Comment:
removed the `if file exists` because that check is done at the entry of the function but also has an `|| base64` check attached to it. (see instrument_LINST class)